src/ucp/proto/proto_debug.c

```diff
@@ -544,8 +544,9 @@ void ucp_proto_perf_node_deref(ucp_proto_perf_node_t **perf_node_p)
     --perf_node->refcount;
     if (perf_node->refcount == 0) {
         ucp_proto_perf_node_free(perf_node);
-        *perf_node_p = NULL;
     }
+
+    *perf_node_p = NULL;
```

Comment:
do we need to pass `**` as a parameter now, when we always set `*perf_node_p = NULL`?

Reply:
yes, in order to set it to NULL.
the caller can also set it to NULL, but IMO it would be safer to enforce it

Reply:
yes, it's different because perf tree nodes point to each other (unlike all other objects we have) so harder to track use-after-free issues